### DISCUSSION TITLE
Add slider end miss visual judgments to argon pro skin

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#25968] [Ideas] [answered]
I don't know if it was intended, but currently the argon pro don't have the visual judgments for slider end misses that was added on today release (2023.1221.0). I think it would be good if there was a option to have it enabled on argon pro because I feel like this is a useful visual feedback.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/pull/25972
